rockcraft/extensions/flask_framework.py

```diff
@@ -198,15 +191,21 @@ def _gen_new_parts(self) -> Dict[str, Any]:
             dependencies_part_name: self._merge_existing_part(
                 dependencies_part_name, dependencies_part
             ),
-            install_app_part_name: self._merge_existing_part(
-                install_app_part_name, install_app_part
-            ),
+            install_app_part_name: {
+                "prime": [
+                    "flask/app/app.py",
```

Comment:
The `app` directory should also be here please

Reply:
Added, thanks!